inspec/a2-api-integration/controls/iam_v2.rb

```diff
@@ -750,4 +750,285 @@
       expect(resp.parsed_response_body[:user_ids]).to eq([user])
     end
   end
+
+  describe "v2beta project rules API" do
+    before(:all) do
+       resp = automate_api_request("/apis/iam/v2beta/projects",
+        http_method: 'POST',
+        request_body: {
+          id: CUSTOM_PROJECT_ID,
+          name: "display name !#$#"
+        }.to_json
+      )
+      expect(resp.http_status).to eq 200
+    end
+
+    after(:all) do
+      resp = automate_api_request("/apis/iam/v2beta/projects/#{CUSTOM_PROJECT_ID}", http_method: 'DELETE')
+      expect(resp.http_status.to_s).to match(/200|404/)
+    end
+
+    context "when the project does not exist" do
+      before(:each) do
+        resp = automate_api_request("/apis/iam/v2beta/projects/#{CUSTOM_PROJECT_ID}", http_method: 'DELETE')
+        expect(resp.http_status.to_s).to match(/200|404/)
+      end
+
+      after(:all) do
+        resp = automate_api_request("/apis/iam/v2beta/projects",
+          http_method: 'POST',
+          request_body: {
+            id: CUSTOM_PROJECT_ID,
+            name: "display name !#$#"
+          }.to_json
+        )
+        expect(resp.http_status).to eq 200
+      end
+
+      describe "GET /iam/v2beta/projects/:id/rules" do
+        it "returns an empty list" do
+          resp = automate_api_request("/apis/iam/v2beta/projects/project-not-found/rules")
+          expect(resp.http_status).to eq 200
+          expect(resp.parsed_response_body[:rules]).to eq([])
+        end
+      end
+
+      describe "GET /iam/v2beta/rules" do
+        it "returns an empty list" do
+          resp = automate_api_request("/apis/iam/v2beta/rules")
+          expect(resp.http_status).to eq 200
+          expect(resp.parsed_response_body[:rules]).to eq([])
+        end
+      end
+    end
+
+    context "when there are no rules but the project exists" do
```

Comment:
The way these tests are organized (with a context block) is different than above-- let's stick with one if possible.

I'm also not convinced we need this many describe blocks but I'd have to look for a bit to see if there's a better option.